citi-onboarding/CESC-client
# (feature):navbar_section

## What I did

- Created the navbar section

## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout feature/navbar`
   1. `git pull origin feature/navbar`
   1. `yarn install`
   1. `yarn start`
   
1. In your browser go to http://localhost:3000/<route>

1. In VScode open `pages - navbar`

1. Things to do during the code:

   - Open insomnia and postbird to test the requisitions needed

1. Read the changed files section on GitHub

1. Test the things that you think are worth testing, even the ones that are not in this description
   - Make sure the navbar is similar to the mockup
   - Make sure the navbar is fixed to the top while scrolling down
